doc/Dataplane Code Gen/DataplaneCodeGen_Quickstart_Tutorial.md

```diff
@@ -0,0 +1,185 @@
+- Delete the `swagger` folder in the `Azure.Template.DataplaneCodeGen` project.
+
+#### 3. Rename the solution and projects
+
+In Visual Studio:
+
+- Rename the `Azure.Template.DataplaneCodeGen` project to `Azure.<your-sdk-name>`.
+- Rename the `Azure.Template.DataplaneCodeGen` solution to `Azure.<your-sdk-name>`.
+- Remove the `Azure.Template.DataplaneCodeGen.Tests` project from the solution.
+
+**Note**: `Azure.<your-sdk-name>` is the Azure SDK package name.
+
+**Learn more:** see the [.NET Guidelines on Namespace Naming](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespace-naming) for more information on naming Azure SDK packages. This name diverges from approved Azure SDK namespaces, since it is intended to be used just for this tutorial.
+
+#### 4. Save and commit your changes to your branch
+
+This will make it easier to see the changes you make when you generate code in the clean solution.
+
+```
+% git add -A
+% git commit -m "initial commit for <service-name>"
+```
+
+### Create the generated layer
+
+In this section, we'll create a generated API layer built on Azure Core. We'll use a small swagger file containing a REST API definition for Azure Messaging WebPubSub.
+
+**Learn more:** the [autorest.csharp README](https://github.com/Azure/autorest.csharp#setup) has great samples showing how to customize the API in the generated code. Explore this further as you design APIs for your own service.
+
+#### 1. Set the Swagger File
+
+Open autorest.md in your `Azure.<your-sdk-name>` project. In the input-file definition, replace the file path with your swagger link, for example: [WebPubSub swagger link](https://github.com/Azure/azure-rest-api-specs/blob/39c7d63c21b9a29efe3907d9b949d1c77b021907/specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json). Update a namespace definition to set the namespace C# generator will use when creating your types. Update the AAD token credential(for AAD token, refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template-LLC/Azure.Template.LLC/src/autorest.md#autorest-configuration) example) or Azure key credential (for Azure Key credential, refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/webpubsub/Azure.Messaging.WebPubSub/src/autorest.md#swagger-sources) example). Save your changes.
+
+Your `autorest.md` file should now look like:
+
+````md
+# `Azure.<your-sdk-name>` Code Generation
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+input-file:
+- <swagger-file-link>
+namespace: <namespace>
+public-clients: true
+low-level-client: true
+security: AADToken
+security-scopes: <AAD-token-scope>
+```
+````
+
+**Learn more:** see the [.NET Guidelines for Namespaces](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespace-naming) to learn more about namespaces in the Azure SDK.
```

Comment:
Let's link to a reference for the readme.md instead of the namespace guidelines here.